.circleci/config.yml

```diff
@@ -0,0 +1,126 @@
+version: 2.1
+
+orbs:
+  kube-orb: circleci/kubernetes@1.3.1
+  helm: circleci/helm@2.0.1
+
+
+commands:
+  build_and_push:
+    description: Build and push Docker image
+    parameters:
+      image:
+        type: string
+      root:
+        type: string
+    steps:
+      - run:
+          name: Build and push Docker image
+          command: |
+            ./scripts/dockerize-and-push.sh freeverseio/<< parameters.image >> << parameters.root >>
+  install_helmfile:
+    description: Install helmfile
+    steps:
+      - run:
+          name: Install helmfile
+          command: ./scripts/install-helmfile.sh
+
+jobs:
+  go_test:
+    docker:
+      - image: cimg/go:1.21
+    steps:
+      - checkout
+      - run:
+          name: Run go tests
+          command: |
+            cd go && go test -race ./...
+  dockerize:
+    docker:
+      - image: cimg/base:2023.09
+    parameters:
+      image:
+        type: string
+      root:
+        type: string
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - build_and_push:
+          image: << parameters.image >>
+          root: << parameters.root >>
+  lint:
+    docker:
+      - image: golangci/golangci-lint:v1.54.2
+    steps:
+      - checkout
+      - run:
+          name: Run golangci-lint
+          command: |
+            cd go && golangci-lint run
+  deployment_lint:
+    docker:
+      - image: cimg/base:2023.09
+    steps:
+      - checkout
+      - helm/install-helm-client:
+          version: v3.11.0
```

Comment:
Ops. I forgot to mention that you can update this to install the latest version, v3.13.0 https://github.com/helm/helm/releases/tag/v3.13.0

Reply:
Resolving conversation as the upgrade has been performed